x-pack/plugin/sql/qa/server/src/main/java/org/elasticsearch/xpack/sql/qa/FieldExtractorTestCase.java

```diff
@@ -74,10 +74,10 @@ public void testTextField() throws IOException {
      *    }
      */
     public void testKeywordField() throws IOException {
+        String query = "SELECT keyword_field FROM test";
```

Comment:
(nice, prepares the reader for what to expect.)

Reply:
@bpintea regarding `noSource()`, agree, it's used once. As part of @costin's review I already removed `noSource()` but didn't commit the change just yet.